src/lib/models/styles.ts

```diff
@@ -56,4 +56,4 @@ export type Color = 'transparent' | 'blue-100'
 | 'yellow-700'
 | 'yellow-900';
 
-export type Size = 0 | 8 | 16 | 24 | 32 | 40 | 48 | 56 | 64 | 72 | 80 | 88 | 96;
\ No newline at end of file
+export type Size = 0 | 8 | 16 | 20 | 24 | 32 | 40 | 48 | 56 | 64 | 72 | 80 | 88 | 96;
```

Comment:
just curious why we are breaking 8x scales rule. was it a design decision?

Reply:
I'm syncing with Andrey today and will check why this is happening, I'll post the feedback here to keep us informed

Reply:
I've seen that design is using more and more smaller units. E.g. `4px` or `12px`.

For me, it makes sense to have finer control on smaller amounts, to let designers have more flexibility tweaking small pieces of UI. So I don't see a problem with adding `20`.

I'd go as far as adding `4` and `12` to the list of allowed sizes and share with Design that this is the list we should stick to.
```suggestion
export type Size = 0 | 4 | 8 | 12 | 16 | 20 | 24 | 32 | 40 | 48 | 56 | 64 | 72 | 80 | 88 | 96;
```

Reply:
totally agreed, i don't have problem breaking 8x either. i think we could even ask designers for a list of common scales, align our code to the list and ask them to stick with it.

Reply:
Maybe it's time to start exploring design tokens, so we are totally aligned between Figma <-> code 👀

Reply:
Talked to Andrey and he also agrees to breaking into 4 in the smaller cases (4, 12, 20 and maybe 28).
For now this will only apply to icons but eventually allow us to margin, padding and so on too (once we get something like [PurgeCSS](https://purgecss.com/) working)

Reply:
And yes to design tokens from figma!!!! 🎉